packages/vite/src/node/plugins/worker.ts

```diff
@@ -262,6 +262,9 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
     },
 
     load: {
+      filter: {
+        id: workerOrSharedWorkerRE,
+      },
```

Comment:
```suggestion
      filter: { id: workerOrSharedWorkerRE },
```
Tweaking a bit to make pulling this change to rolldown-vite a bit easier.